core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -279,13 +279,13 @@ class RequestSendThread(val controllerId: Int,
 
 }
 
-class ControllerBrokerRequestBatch(controller: KafkaController) extends  Logging {
+class ControllerBrokerRequestBatch(controller: KafkaController, eventManager: ControllerEventManager) extends  Logging {
```

Comment:
@ijuma Thanks for the explanation. Do you prefer to include `eventManager` explicitly in the constructor of `ControllerBrokerRequestBatch`, or is it OK to reference `eventManager` via `KafkaController`?